lms/templates/main.html

```diff
@@ -101,9 +101,8 @@
     else:
         header_file = microsite.get_template_path('navigation.html')
 
-    google_analytics_file = microsite.get_template_path('google_analytics.html')
```

Comment:
@asadiqbal08 -- in this particular case we need to continue supporting Google Analytics for the many Open edX operators/installations in the world, who will not be switching to Google Tag Manager.  So we'll need to keep the reference to google_analytics.html and add an override to use google_tag_manager.html instead.